packages/modules-sdk/src/definitions.ts

```diff
@@ -75,7 +75,7 @@ export const ModulesDefinition: { [key: string | Modules]: ModuleDefinition } =
       isRequired: false,
       canOverride: true,
       isQueryable: true,
-      dependencies: [],
+      dependencies: ["eventBusModuleService"],
```

Comment:
todo: can we use the definition registration name instead?

Reply:
<img width="477" alt="Screenshot 2023-08-02 at 18 03 19" src="https://github.com/medusajs/medusa/assets/5105988/7afadd3f-4e70-4b52-936f-716559db1027">
This is the registration name tho?

Reply:
I mean moduleDefinitions[Modules.Product].registrationName